apps/astro/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "astro",
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "astro": "astro",
+    "build": "astro build",
+    "dev": "astro dev",
+    "preview": "astro preview",
+    "start": "astro dev"
+  },
+  "dependencies": {
+    "@astrojs/mdx": "^1.1.5",
+    "@astrojs/rss": "^3.0.0",
+    "@astrojs/sitemap": "^3.0.3",
+    "astro": "^3.6.3"
```

Nit: This is just example project, but Astro 4 came out a week ago, it wouldn't hurt [upgrading the dependencies](https://github.com/withastro/astro/blob/main/examples/blog/package.json#L14-L17)
```suggestion
    "@astrojs/mdx": "^2.0.1",
    "@astrojs/rss": "^4.0.1",
    "@astrojs/sitemap": "^3.0.3",
    "astro": "^4.0.4"
```